### PR TITLE
[RLlib] minor fix for get_init_state annotation in torch

### PR DESCRIPTION
**Pull request by kouroshHakha**
## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by avnishn — APPROVED** [review]

**Comment by kouroshHakha** [comment]
@sven1977 Could you merge please?

**Review by sven1977 — APPROVED** [review]
Thanks @kouroshHakha .
Sorry, just one nit. We should change `TensorType` to `Union[np.ndarray, tf.Tensor, torch.Tensor]`.
Right now it's `Any`. Then this line should just be:
`... -> List[TensorType]`

File: rllib/utils/typing.py [hunk 1]
```diff
@@ -29,7 +30,7 @@
 
 # Represents a generic tensor type.
 # This could be an np.ndarray, tf.Tensor, or a torch.Tensor.
-TensorType = Any
+TensorType = Union[np.array, tf.Tensor, torch.Tensor]
```
Comment (sven1977) [inline rllib/utils/typing.py:33]:
```suggestion
TensorType = Union[np.array, "tf.Tensor", "torch.Tensor"]
```

**Comment by sven1977** [comment]
One more LINT fix (already done) :) ...

File: rllib/utils/typing.py (hunk 1 above)
Comment (kouroshHakha) [reply rllib/utils/typing.py]: thanks

**Pull request merged**